### PR TITLE
[Feature] Enterprise Edition support

### PR DESCRIPTION
**Pull request by ajanikow**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -95,11 +97,13 @@ endif
 
 BASEUBIIMAGE ?= registry.access.redhat.com/ubi8/ubi-minimal:8.4
 
+OPERATORIMAGENAME ?= $(REPONAME)
```
Comment (ewoutp) [inline Makefile:100]: Is this overwritten for enterprise?

**Review by Robert-Stam — APPROVED** [review]
LGTM

File: Makefile (hunk 1 above)
Comment (ajanikow) [reply Makefile:100]: Yes, release-manager will set it for enterprise to (kube-arangodb-enterprise or kube-arangodb-enterprise-preview)

Comment (ajanikow) [reply Makefile:100]: In make we just set default value to be able to build everything using old methods

**Review by ewoutp — APPROVED** [review]
LGTM

**Pull request merged**